raiden/ui/cli.py

```diff
@@ -965,8 +965,10 @@ def _run_app(self):
             console = Console(app_)
             console.start()
 
-        # spawning a thread to handle the version checking
+        # spawn a greenlet to handle the version checking
         gevent.spawn(check_version)
+        # spawn a greenlet to handle the gas escrow check
+        gevent.spawn(check_gas_escrow, app_.raiden)
```

Comment:
We may have more periodical checks like this in the future. Not necessarilly something you need to address in this PR but can't we just have a periodic checker greenlet which will do all these checks in the same periods of time and then spare us the additional always-running greenlets?

Reply:
I agree, the problem I see is when we want different check periods. I let thee gas reserve check run every hour, while the version check runs ever 3h.

Reply:
In this case we can just do a more frequent version check. But yeah the problem is if we really would need different periods for different checks.